apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PathExpectationSamples.kt

```diff
@@ -244,11 +246,16 @@ class PathExpectationSamples {
         val extension = "txt"
         val dir = tempDir.newFile("test_file.$extension")
 
-        expect(dir).extension.notToBeEmpty().toEqual(extension)
+        expect(dir).extension // subject is now of type String (actually "txt")
+            .notToBeEmpty()
+            .toEqual(extension)
+            .notToEndWith("jpg")
 
         fails {
-            val invalidExtension = "txtt"
-            expect(dir).extension.toEqual(invalidExtension)
+            expect(dir).extension // subject is now of type String (actually "txt")
+                .toEqual("txtt") // fails because it doesn't equal to "txtt"
+                .toEndWith("jpg") // not reported
+            //                              use `.extension` if you want that all expectations are evaluated
```

Comment:
Can you please align the comments, makes it a bit more pleasant to read (also further below)
```suggestion
            expect(dir).extension  // subject is now of type String (actually "txt")
                .toEqual("txtt")   // fails because it doesn't equal to "txtt"
                .toEndWith("jpg")  // not reported
            //                        use `.extension` if you want that all expectations are evaluated
```